cpp/ql/src/Summary/LinesOfCode.ql

```diff
@@ -0,0 +1,11 @@
+/**
+ * @id cpp/summary/lines-of-code
+ * @name Total lines of C/C++ code in the database.
+ * @description The total number of lines of C/C++ code across all files, including system headers, libraries and auto-generated files. This is a useful metric of the size of a database. Lines of code are all lines in a file that was seen during the build that contain code, i.e. are not whitespace or comments.
```

Comment:
```suggestion
 * @description The total number of lines of C/C++ code across all files, including system headers, libraries, and auto-generated files. This is a useful metric of the size of a database. Lines of code are all lines in a file that was seen during the build that contain code, i.e. are not whitespace or comments.
```

We like using the Oxford comma.

The second sentence has some ambiguity issues around "the build that contain code", but I'll leave it to the docs team to clear that up. We used to avoid "i.e." under the Semmle style guide, but I don't know what we do these days as GitHub.